app/middleware/error-handler.js

```diff
@@ -60,5 +62,6 @@ module.exports = function errorHandler (err, req, res, next) {
   logger.error(`Unhandled error caught: ${err.message}`, {
     stack: err.stack
   })
+  Sentry.captureException(err)
```

Comment:
Along with capturing the exception with Sentry here, is there a reflective part to remove it from the existing method of sending an event to sentry on `error` level logs? 

An example of the design consideration there is that the line of code above registers a `logger.entry` which I _think_ (would need confirming) will be treated as a unique event by Sentry. 

Code that sets that up https://github.com/alphagov/pay-selfservice/blob/master/app/utils/logger.js#L29